docs/conf.py

```diff
@@ -253,3 +260,85 @@
 
 # How to display URL addresses: 'footnote', 'no', or 'inline'.
 # texinfo_show_urls = 'footnote'
+
+schema_top_docs = {
+    'abstracts': 'Abstracts for a conference or workshop. This is generally public information\n\n',
+    'assignments': 'Information about assignments for classes.\n\n',
+    'blank': '\n',
```

Comment:
not needed since it isn't a real schema and it shouldn't be generated.

Reply:
you actually can probably just remove the blank.rst file since it is no longer needed.